dali/kernels/slice/slice_gpu.cuh

```diff
@@ -72,8 +72,14 @@ struct SliceBlockDesc {
   uint64_t size;
 };
 
+#if CUDA_VERSION >= 11000
```

Comment:
```suggestion
#if CUDA_VERSION >= 11200
```
11.2 is the first compiler that doesn't do it.
Also, 10.1 seems unaffected, too - but I don't think we care.